examples/AlwaysMints.purs

```diff
@@ -0,0 +1,141 @@
+-- | This module demonstrates how the `Contract` interface can be used to build,
+-- | balance, and submit a smart-contract transaction. It creates a transaction
+-- | that mints a value using the `AlwaysSucceeds` policy
+-- |
+-- | * Prerequisites
+-- |   - A Chromium-based browser (for Nami compatibility)
+--
+-- |   - A Nami wallet funded with test Ada ("tAda") and collateral set, If you need
+-- |     tAda, visit https://testnets.cardano.org/en/testnets/cardano/tools/faucet/
+--
+-- | * How to run
+--
+-- |   The `Contract` interface requires several external services to be running.
+-- |   From the repository root, run `nix run .#ctl-runtime` to launch all
```

Comment:
Let's not include particular commands in the docs. Referring to the README is better for large projects to prevent documentation rot. Also, there are multiple ways to run the services, and this one might not be the best for the reader.